src/common/list.c

```diff
@@ -393,7 +393,7 @@ struct oscap_htable *oscap_htable_new1(oscap_compare_func cmp, size_t hsize)
 		return NULL;
 	t->hsize = hsize;
 	t->itemcount = 0;
-	t->table = calloc(hsize, sizeof(struct oscap_list_item *));
+	t->table = calloc(hsize, sizeof(struct oscap_htable_item *));
```

Comment:
wow, just wow

:D

Reply:
It didn't cause problems because size of pointer to a structure is same as size of pointer to a different structure.